src/vm/functions/assets.rs

```diff
@@ -59,6 +59,7 @@ pub fn special_stx_transfer(args: &[SymbolicExpression],
         env.global_context.database.set_account_stx_balance(&to,   final_to_bal);
 
         env.global_context.log_stx_transfer(&from, amount)?;
+        env.register_stx_transfer_event(from.clone(), to.clone(), amount)?;
```

Comment:
Stx transfer events are not being generated from TokenTransfer transaction types. Is this intended?

Reply:
Can you please provide your toml file and a dump of your transaction so that I can investigate?
It's working with: 

```
[[events_observer]]
port = 8080
address = "127.0.0.1"
events_keys = [
    "*",
]
[[events_observer]]
port = 8081
address = "127.0.0.1"
events_keys = [
    "stx"
]

[[mstx_balance]]
address = "ST2T9J9B3X0087W3FZKWFVMAJS1KX1PAPWM2PJ8FP"
amount = 100000

```

and ```80000000000400b4992563e80083f06ffcf8fdd152c867d0d956e5000000000000000000000000000000000000a6d92d8434b625bbf8730aeb8f9b0b9354995883e64dc4ebf69fe46defb4227c335ec88b66a7332fbe79c642dbd751dab7fc806b9b6445be62348838116919a8030200000000001625a2a51cf0712a9d228e2788e2fe7acf8917ec8100000000000003e800000000000000000000000000000000000000000000000000000000000000000000```

Thanks!

Reply:
nvm! This was something wrong on my end. It's working as expected!